.eslintrc.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+  },
+  extends: "@guardian/eslint-config-typescript",
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    ecmaVersion: 12,
+    tsconfigRootDir: __dirname,
+    project: ["./tsconfig.eslint.json"],
+  },
+  plugins: ["@typescript-eslint", "custom-rules"],
+  rules: {
+    "@typescript-eslint/no-inferrable-types": 0,
+    "import/no-namespace": 2,
+    "@typescript-eslint/no-unsafe-assignment": 1,
+    "@typescript-eslint/no-explicit-any": 1,
```

Comment:
These rules have been manually set to `warning` (`1`) as they would otherwise be a lot of work to fix! We can address them in future PRs, as they are still surfaced as warnings in the console when running `npm run lint`.